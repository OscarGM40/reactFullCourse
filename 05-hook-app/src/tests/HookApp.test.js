import { shallow } from 'enzyme';
import React from 'react';
import HookApp from '../HookApp'

describe('Pruebas en HookApp', () => {
   
   test('Debe de hacer match con la snapshot', () => {
      const wrapper =shallow(< HookApp />);

      expect(wrapper).toMatchSnapshot();


   })
   
})
