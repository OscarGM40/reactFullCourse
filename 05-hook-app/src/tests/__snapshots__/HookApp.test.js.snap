// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en HookApp Debe de hacer match con la snapshot 1`] = `
<div>
  <h1>
    Hola mundo
  </h1>
</div>
`;
